### ISSUE TITLE
[Insights team: Data Futures Lab] Data Stewardship Ecosystem and Demand study - PDF and not Publication-related support

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/foundation.mozilla.org#6132]
### Background:
This is the part II of the [research we already published on our site](https://foundation.mozilla.org/en/initiatives/data-futures/). 
The title of the research is "Data Stewardship Ecosystem and Demand study". Sabrina is working with the Insights Team (work being tracked on [this ticket](https://github.com/MozillaFoundation/Design/issues/751)) to support them with Publications guidance.

### To do:
SITE
- [x] Insights team can populate their page with content **Responsible:** Insights team STATUS: waiting on the team to populate content
- [x] Once Publications is release (this study and previous one), add them to [Data Futures Lab Hub](https://foundation.mozilla.org/en/data-futures-lab/).

PDF 
- [x] Add content to Mozilla Foundation Doc template **Responsible:** Insights team 
- [x] Do a design review, connect PDF with site **Responsible:** Taís 

SITEMAP
- [x] Update line 38 (second tab) of [this spreadsheet ](https://docs.google.com/spreadsheets/d/184z7LnHIghXDp2NIns4T8zlpBDAdaRFuPMUDX7ijHYY/edit?usp=sharing)with accurate URL

INSIGHTS HUB
- [x] Surface this study in the [Insights Hub](https://foundation.mozilla.org/en/insights/) - an idea was to display this and the previous study - also mentioning they were both part of the Data Futures Lab initiative. [Example here](https://www.figma.com/file/AABYnCVDRDfyMSyyTYiGAH/URL-plan-Mozilla-Research?node-id=141%3A0). Need to connect with Kasia and team to see how this can work.
This study is already inside the previous study (same Publication) so it is already present on the Insights team Hub

### Deadline
**Publication planned for March 30, we hope to have the text frozen by March 24**. 

### References:
**Airtable link:** https://airtable.com/tbljPmOCkK9JShaTS/viw9sdUJiIjCYpn0Y/rec0Abfjqo93TEU07?blocks=hide
**Draft copy:** https://docs.google.com/document/d/1D1nVFMC9WCWHq8nVyP3AbqGfde6U-z2JpL36_HRb1vY/edit#heading=h.t4ihadghehzo
**Epic ticket:**  #6047
**Design needs (doc prepared by Insights team)**: https://docs.google.com/document/d/1Bf9N7p5qvza7HFq7oUDUcdBtsVae6RVEDDsI6Fj-63c/edit?usp=sharing

**Comment by kristinashu** [comment]
Seems like this is blocked until the start of next sprint, is that right?

**Comment by taisdesouzalessa** [comment]
Yep. I'll move this to next sprint. Haven't heard anything about it during this one.

**Comment by kristinashu** [comment]
Any update on this timeline?

**Comment by jamilasnell** [comment]
"The Data Futures part will be moved this week (first part) so we can plug it into the Data futures Lab website that will be launched at Mozfest. And then the second part will be probably moved in the week of the 15th so we can launch on March 24."

**Comment by taisdesouzalessa** [comment]
From Eeeva: "I just did my first draft of the [ecosystem PDF](https://docs.google.com/document/d/1yPokbfI-3TEiUl6hcRvq3XGcopHfdML--IxSM6JklV4/edit?usp=sharing) and wanted to share it with you. I am going to tag you in the doc with a few specific questions, just wanted to give you a heads up. Any and all feedback is welcome. I tried to stay close to the style we had used for the previous PDFs."

**Comment by taisdesouzalessa** [comment]
**PDF tweaks:**
1. Created an image for the cover
<img width="812" alt="image" src="https://user-images.githubusercontent.com/22157921/112543499-18195180-8d73-11eb-9cdd-f980fc922c1f.png">

2. Improved quote component to be more legible:
<img width="712" alt="image" src="https://user-images.githubusercontent.com/22157921/112543649-49921d00-8d73-11eb-8747-8250be5ba5db.png">

3. Improved "Spotlight" section (please don't consider the yellow rectangle, it is just a highlight from Google Doc - for a comment we have there)
<img width="660" alt="image" src="https://user-images.githubusercontent.com/22157921/112543756-6d556300-8d73-11eb-9e21-262ad763edb7.png">

I went as far as I could on this PDF and I don't think we will have any major work to do after this. If there is a significant lift, I will open another ticket. Closing this ticket.
